### ISSUE TITLE
Support for Nuvoton NUC029LAN

### ISSUE THREAD

**Issue by tremez** [stawel/cheali-charger#248]
Just got a new animal with this "new?" chip.
Is it supported ? Looks like it is same Cortex-M0 as 0519M

**Comment by stawel** [comment]
unfortunately it's not supported

**Comment by Stormwing01** [comment]
When Will it be supported?

**Comment by flaksirus** [comment]
@Stormwing01 I don't think it will be ever supported - it's absolutely different mcu with different architecture, and obviously there should be absolutely different schematic, so it would really hard to port this firmware to nuvoton analog.

**Comment by pampyras** [comment]
Hi. Any progress on nuc029lan support?

**Comment by diggit** [comment]
There is no active developer at this moment. I guess, that contributions are welcome. 

Where did you get charger with this MCU?

**Comment by pampyras** [comment]
From Aliexpress. I have made it semi working by adapting code, but still need some improvents to make it usable.

**Comment by shadowlab** [comment]
https://github.com/stawel/cheali-charger/issues/241#issuecomment-855119485

I've made a post in another thread regarding this CPU and charger. Hopefully, this will work OK.
